### DISCUSSION TITLE
Find out what users do when they open the app

### DISCUSSION THREAD

**Discussion by tbertels** [openfoodfacts/openfoodfacts-androidapp#4437] [General]
#127 added an option to start scan on app open.

But do most users open the app to scan a product?
If that's the case, this option could be enabled by default.

- The app could send data back about the first action the user does, so that it would be possible to decide if it should be enabled by default for all users.

- The app could suggest to enable automatically that option if most of the time the user opens the app to scan a product.

- These two previous could be combined: send back to the server the value of the option after enough app use.

**Comment by CloCkWeRX** [comment]
Annecdata; I use this for:

Scanning / contributing to open food facts *after* I've made a purchase. Currently, its just slightly too slow to scan, and if the product is unknown, I may not want to add it while shopping.

Ideally, I could use this to scan as I'm throwing out something, and *add it to a shopping list* - google's, home assistant, or something internal.
